dirsrvtests/tests/suites/paged_results/paged_results_test.py

```diff
@@ -237,6 +239,10 @@ def paged_search(conn, suffix, controls, search_flt, searchreq_attrlist, abandon
     msgid = conn.search_ext(suffix, ldap.SCOPE_SUBTREE, search_flt, searchreq_attrlist, serverctrls=controls)
     log.info('Getting page %d' % (pages,))
     while True:
+        if avoid_race_con:
+            # This should only be true when calling from test_multi_suffix_search() to avoid a race condition
+            # with multiple queries.
+            sleep(1)
         try:
             rtype, rdata, rmsgid, rctrls = conn.result3(msgid, timeout=0.001)
```

Comment:
What about increasing the result3() timeout?  Would that help?

Reply:
Raising the timeout doesn't seem to make much of a difference, unfortunately. I previously raised it to 1, and then 10, and the test still failed due to log order. I'm thinking progier389's idea of reordering logs by opid might be the best solution here.

Reply:
Gotcha, was just throwing out ideas :-)